client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -37,12 +37,22 @@ class DesignTypeWithStoreStep extends Component {
 	getChoices() {
 		const { translate } = this.props;
 
-		return [
-			{ type: 'blog', label: translate( 'A list of my latest posts' ), image: <BlogImage /> },
-			{ type: 'page', label: translate( 'A welcome page for my site' ), image: <PageImage /> },
-			{ type: 'grid', label: translate( 'A grid of my latest posts' ), image: <GridImage /> },
-			{ type: 'store', label: translate( 'An online store' ), image: <StoreImage /> },
-		];
+		switch ( abtest( 'signupCopy' ) ) {
```

Comment:
You probably hit our [`no-else-return` eslint rule](http://eslint.org/docs/rules/no-else-return) :)
```
if ( condition ) {
   return x;
} else {
   return y;
}
```
is not allowed (the `else` branch is not needed, because of the `return` in the `if` branch). So, it should be:
```
if ( condition ) {
   return x;
}

return y;
```
Hope that makes sense (the documentation I've linked above might explain it better :D).